cpp/ql/lib/change-notes/2022-02-24-structured-bindings-in-ir.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Structured bindings are handled in the IR translation.
```

Comment:
What does this mean for a user? Or are these change note not intended for users?

Reply:
The IR is visible to the user. Does that answer your question.

Reply:
Maybe we should also write something along the lines of: `As a result, many queries now support structured bindings.`?

Reply:
Done in a slightly different form, so please check.

Reply:
Looks good. Thanks!